libs/blocks/video/video.js

```diff
@@ -1,10 +1,10 @@
-import { createIntersectionObserver, getConfig } from '../../utils/utils.js';
-import { applyHoverPlay, getVideoAttrs, applyInViewPortPlay } from '../../utils/decorate.js';
+import { getConfig } from '../../utils/utils.js';
+import { decorateAnchorVideo } from '../../utils/decorate.js';
 
-const ROOT_MARGIN = 1000;
-
-const loadVideo = (a) => {
-  const { pathname, hash, dataset } = a;
+export default function init(a) {
+  a.classList.add('hide-video');
+  if (!a.parentNode) return;
```

Comment:
In this scenario, we'd be left with an anchor with `visibility: hidden;` style, is this the expected outcome? Would we want to remove the anchor, which would normally happen in `decorateAnchorVideo`?

Reply:
Can this even ever happen?